.github/workflows/verify.yml

```diff
@@ -0,0 +1,91 @@
+# Copyright 2023 DSR Corporation, Denver, Colorado.
+# https://www.dsr-corporation.com
+# SPDX-License-Identifier: Apache-2.0
+
+name: Verify
+
+on:
+  pull_request:
+    branches: [main]
+
+
+jobs:
+
+  release-ready:
```

Comment:
Should this job be "needed" by others?  It's just sitting on it's own at the moment.